serverless/knative_serving/serverless-traffic-management.adoc

```diff
@@ -0,0 +1,27 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-traffic-management"]
+= Traffic management
+include::modules/common-attributes.adoc[]
+:context: serverless-traffic-management
+
+toc::[]
+
+// TODO later: consider splitting this into a conceptual module about revisions; would need to plan where these conceptual topics would be included in docs
+A _revision_ is a point-in-time snapshot of the code and configuration for each modification made to a Knative service. Each time the configuration of a service is updated, a new revision for the service is created. Revisions are immutable objects and can be retained for as long as they are required or used. Knative Serving revisions can be automatically scaled up and down according to incoming traffic.
+
+You can manage traffic routing to different revisions of a Knative service by modifying the `traffic` spec of the service resource.
+
+image::knative-service-architecture.png[Knative service architecture]
+// placeholder image until we get a nice diagram to replace this
+
+// routing basics - YAML examples
+include::modules/serverless-traffic-routing-examples.adoc[leveloffset=+1]
+// ODC
+include::modules/odc-splitting-traffic-between-revisions-using-developer-perspective.adoc[leveloffset=+1]
+// kn CLI
+include::modules/serverless-traffic-management-kn.adoc[leveloffset=+1]
+include::modules/serverless-workflow-traffic-splitting-flags.adoc[leveloffset=+2]
```

Comment:
I guess it isn't part of the PR, but in title for this file "Traffic splitting flags by using the Knative CLI" seems like incongruous with the other titles beneath the Knative syntax reference for traffic management.

Cosmetically, "Traffic splitting flags" (or "Traffic management flags") might have more visual appeal.  I don't have enough SEO expertise to know if that's a step backward or inconsequential.